bundles/org.openhab.core/src/main/java/org/openhab/core/internal/scheduler/CronAdjuster.java

```diff
@@ -430,12 +445,6 @@ private static boolean checkMaxYear(Temporal temporal) {
                     cronExpression, chronoField, r[1], max));
 
         }
```

Comment:
Would be really nice to get rid of this empty line in front of the closing bracket.

```suggestion
                    cronExpression, chronoField, r[1], max));
        }
```